tests/policy/test.sh

```diff
@@ -75,7 +100,11 @@ rlJournalStart
             "Verify that test has been modified" \
             "$(yq -cSr '.[] | .test' $run/default/plan/discover/tests.yaml)" \
             "bash -c 'echo \"Spiked test.\"; /bin/true'"
+    rlPhaseEnd
 
+    rlPhaseStartTest "Invalid keys"
+        rlRun -s "tmt -vv test export --policy ../policies/test/invalid.yaml /basic" 2
+        rlAssertGrep "Could not find field 'script' in class '/basic'." $rlRun_LOG
```

Comment:
`/basic` is not a class name. Seems that `container.__class__.__name__` should be used in the exception instead?

Reply:
I agree, it's not perfect, but I would prefer not to touch it in this PR. I have no idea what else will break down. It deserves an improvement, but I don't want to open yet another can of worms in this point of Friday afternoon.